.github/workflows/build.yml

```diff
@@ -188,7 +188,7 @@ jobs:
           extra: ios
           cc: clang
           cxx: clang++
-          args: ./b.sh --ios
+          args: chmod +x b-ios.sh && ./b-ios.sh
```

Comment:
the `chmod` is odd IMHO, you should add the script to the repo with the correct permissions (i.e. with the execute bit)

Reply:
I'll fix the permission post-merge for simplicity.